config/preinitializer.rb

```diff
@@ -3,3 +3,6 @@
   $req_log_path = File.join(File.dirname(__FILE__), %w(.. log))
   require 'require_with_logging'
 end
+
+require File.join(__dir__, "..", "lib", "code_coverage")
+CodeCoverage.run_hook
```

Comment:
So I would need to provide in the file $railsroot/lib/code_coverage.rb a CodeCoverage class with a run_hook() method?

Reply:
Oh wait code_coverage.rb is your file, it loads the file I'm already putting there, coverage_hook.rb?

Reply:
Yeah, see the comments below... just dump your script that starts/restarts code coverage in config/coverage_hook.rb and it will be loaded when the server starts and when new forked workers are started.  Let me know how it goes.

Reply:
@jamesooden If you're not aware, you can generate a patch file from any PR that you can apply to manageiq/cf appliances using `patch -p1 < this.patch` or `git am` if manageiq git enabled appliances.

Just throw `.patch` at the end of the pull request number:
https://github.com/ManageIQ/manageiq/pull/17302.patch